docs/user_guide/model_analyzer.md

```diff
@@ -35,7 +35,9 @@ utilization. The Model Analyzer is specifically useful for characterizing the
 GPU memory requirements for your model under different batching and model
 instance configurations. Once you have this GPU memory usage information you can
 more intelligently decide on how to combine multiple models on the same GPU
-while remaining within the memory capacity of the GPU.
+while remaining within the memory capacity of the GPU. Refer 
+[this guide](https://github.com/triton-inference-server/tutorials/tree/main/Conceptual_Guide/Part_3-optimizing_triton_configuration)
+ for an example.
```

Comment:
1. Remove this here
```suggestion
while remaining within the memory capacity of the GPU.
```

2. Replace "For more information see the [Model Analyzer repository] ..." at the bottom with something like:
```
For more detailed examples and explanations of using Model Analyzer, see:
- [Model Analyzer Conceptual Guide](https://github.com/triton-inference-server/tutorials/tree/main/Conceptual_Guide/Part_3-optimizing_triton_configuration)
- [Maximizing Deep Learning
Inference Performance with NVIDIA Model
Analyzer](https://developer.nvidia.com/blog/maximizing-deep-learning-inference-performance-with-nvidia-model-analyzer)
```


3. Add hyper link on Model Analyzer to the start of this paragraph
```
The Triton [Model Analyzer](https://github.com/triton-inference-server/model_analyzer) is a tool that uses ...
```